feat(profiling): add wizard onboarding docs (#5778)
getsentry/sentry-docs: src/wizard/android/profiling-onboarding/android/1.install.md

---
name: Android
doc_link: https://docs.sentry.io/platforms/android/profiling/
support_level: production
type: language
---

#### Install

> For the Profiling integration to work, you must have the Sentry Android package (minimum version 6.0.0). We offer an automatic (shown below) and manual installation methods using Sentry’s Android Gradle Plugin. Learn more about installation methods in our [full documentation](https://docs.sentry.io/platforms/android/#install).

Reviewer: ```suggestion
For the Profiling integration to work, you must have the Sentry Android package (minimum version 6.0.0). You can choose between automatic and manual installation methods, using Sentry’s Android Gradle Plugin (shown below). Learn more about installation methods in our [full documentation](https://docs.sentry.io/platforms/android/#install).
```
